cl2-component-library/src/components/LateralScrollControls/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useState } from 'react';
+import { useTheme } from 'styled-components';
+import Box from '../Box';
+import Button from '../Button';
+import testEnv from '../../utils/testUtils/testEnv';
+interface Props {
+  containerRef?: React.RefObject<HTMLDivElement>; // Ref of the container with elements which can be laterally scrolled through
+  scrollBtnDistanceMobile?: number; // Lateral scroll distance on arrow click for mobile
+  scrollBtnDistanceDesktop?: number; // Lateral scroll distance on arrow click for desktop
+  isSmallerThanPhone?: boolean;
```

Comment:
I originally tried to use the useBreakpoint hook, but it seems that it can't be used in the component library? Is that intended? :thinking: